devito/archinfo.py

```diff
@@ -258,7 +271,8 @@ def lscpu():
     if output:
         lines = output.decode("utf-8").strip().split('\n')
         mapper = {}
-        for k, v in [tuple(i.split(':')) for i in lines]:
+        # Using split(':', 1) to avoid splitting lines with security issues
```

Comment:
"with security issues" . Can you be more precise? Nobody will understand it

```
# Using split(.....
# Example: ...
```

Reply:
What about:
```
        # Using split(':', 1) to avoid splitting lines where lscpu shows vulnerabilities
        # on some CPUs: https://askubuntu.com/questions/1248273/lscpu-vulnerabilities
```